sdk/src/main/java/io/opentelemetry/sdk/metrics/MeterSdk.java

```diff
@@ -81,4 +84,14 @@ public LabelSetSdk createLabelSet(String... keyValuePairs) {
   public LabelSetSdk createLabelSet(Map<String, String> labels) {
     return LabelSetSdk.create(labels);
   }
+
+  Collection<MetricData> collectAll() {
+    InstrumentRegistry instrumentRegistry = meterSharedState.getInstrumentRegistry();
+    Collection<AbstractInstrument> instruments = instrumentRegistry.getInstruments();
+    ArrayList<MetricData> ret = new ArrayList<>(instruments.size());
```

Comment:
left hand side should just be declared as `List<MetricData>`
I'd also prefer the temporary variable be called `results`, but that's a nitpick.

Reply:
Done.